### PR TITLE
Login with urbit proposal

### PR DESCRIPTION
**Pull request by mikeosborne**

### CODE REVIEW

**Comment by mikeosborne** [comment]
This builds fine on my local repo, but fails here due to broken links in the docs. I'm unsure how to proceed.

File: content/blog/gifts-q1-2021.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+image ="https://media.urbit.org/site/posts/essays/gifts.png"
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/gifts.png">
+
+<br>
+
+Twice a year we distribute address space to those that have made valuable contributions to Urbit. Pervious the rational and our thinking about the role of gifts can be seen in our previous gift announcements [here](https://urbit.org/blog/urbit-grants-and-mid-2019-gifts/) and [here](https://urbit.org/blog/gifts-q3-2020/). without further ado, here are the 2021 Q1 gift recipients (all to receive 1 star):
```
Comment (drbeefsupreme) [inline content/blog/gifts-q1-2021.md:19]: Really just fixing the typo but the sentence seemed awkward anyways so here's a suggested revision.
```suggestion
Twice a year we distribute address space to those that have made valuable contributions to Urbit. We've shared our rationale and thinking about the role of gifts in our previous gift announcements [here](https://urbit.org/blog/urbit-grants-and-mid-2019-gifts/) and [here](https://urbit.org/blog/gifts-q3-2020/).

Without further ado, here are the 2021 Q1 gift recipients (all to receive 1 star):
 ```

**Review by drbeefsupreme — CHANGES_REQUESTED** [review]

File: content/blog/gifts-q1-2021.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Recipients
+
+ - `~nodreb-borrus`: 	https://urbit.me/
+~sitful-hatred	https://subject.network
+~tocrex-holpen	https://twitter.com/tocrexholpen
+~sigryn-habrex	Urbit Chess
+~habsul-rignyr	The Stack Podcast
+~libhut-samwes	The Stack Podcast
+~tapruc-solwer	Bitcoin Wallet Consulting
+~litmus-ritten	cultural acheivement
```
Comment (fischerando) [inline content/blog/gifts-q1-2021.md:31]: typo here - should be "achievement"

**Comment by mikeosborne** [comment]
Closing because I mucked up commits. Dividing commits into separate PRs.

**Pull request closed without merging**